cmake/common/px4_base.cmake

```diff
@@ -332,6 +332,8 @@ function(px4_add_common_flags)
 		-Wunknown-pragmas
 		-Wunused-variable
 
+		-Wno-implicit-fallthrough # set appropriate level and update
+
```

Comment:
I actually like this warning. Better be explicit when fallthrough is intended, like here:
https://github.com/PX4/Firmware/blob/bcceadcb85b293046166ed7898c96b7dbf9acd30/src/drivers/mpu9250/mpu9250_spi.cpp#L168-L184